decidim-core/db/migrate/20180209122819_create_decidim_searchable_rsrcs.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class CreateDecidimSearchableRsrcs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :decidim_searchable_rsrcs do |t|
+      t.text :content_a
+      t.text :content_b
+      t.text :content_c
+      t.text :content_d
+      t.string :locale, null: false
+      t.datetime :datetime
+
+      t.belongs_to :decidim_scope
+      t.belongs_to :decidim_participatory_space, polymorphic: true, index: { name: "idx_decidim_schbl_rsrcs_on_ptcptry_spc_type_and_ptcptry_spc_id" }
```

Comment:
Can you try to find another index name that can be readable, please?

Reply:
index_decidim_searchable_resource_on_participatory_space_type_and_participatory_space_id => index_decidim_searchable_resource_on_pspace_type_and_pspace_id